ops/modules/ecr/main.tf

```diff
@@ -12,12 +12,52 @@ resource "aws_ecr_lifecycle_policy" "remove_old_images" {
     "rules": [
         {
             "rulePriority": 1,
-            "description": "Expire images older than 20 days",
+            "description": "Expire main images that are not the last 50",
             "selection": {
-                "tagStatus": "any",
+                "tagStatus": "tagged",
+                "tagPrefixList": ["main-"],
+                "countType": "imageCountMoreThan",
+                "countNumber": 50
```

Comment:
What is the intuition behind having main > staging > prod counts?

Reply:
Mostly due to how often they get built. Merges to main happen almost daily, to prod anywhere from once every few days to once every 2 weeks. 

The idea is keeping a reasonable amount of images in case rollbacks are needed